accessibility_monitoring_platform/apps/notifications/templates/notifications/task_list.html

```diff
@@ -24,90 +18,121 @@ <h1 class="govuk-heading-xl amp-margin-bottom-30">{{ sitemap.current_platform_pa
+                                <b> View read </b>
+                            {% else %}    
+                                <a href="?type=qa-comment&read=true" class="govuk-link govuk-link--no-visited-state govuk-link--no-underline">View read</a>
+                            {% endif %}
+                        </li>
+                    </ul>
+                {% endif %}
+                {% if type == 'report-approved' %}
+                    <ul class="govuk-list">
+                        <li class="amp-custom-nav-bar amp-margin-right-25">
+                            {% if read == None %}
+                                <b>Report approved ({{ task_type_counts.report_approved }})</b>
+                            {% else %} 
+                                <a href="?type=report-approved" class="govuk-link govuk-link--no-visited-state govuk-link--no-underline">Report approved ({{ task_type_counts.report_approved }})</a>
+                            {% endif %}
+                        </li>
+                        <li class="amp-custom-nav-bar amp-margin-right-25">
+                            {% if read %}
+                                <b> View read </b>
+                            {% else %}    
+                                <a href="?type=report-approved&read=true" class="govuk-link govuk-link--no-visited-state govuk-link--no-underline">View read</a>
+                            {% endif %}
+                        </li>
+                    </ul>
+                {% endif %}
+                {% if type == 'reminder' %}
+                    <ul class="govuk-list">
+                        <li class="amp-custom-nav-bar amp-margin-right-25">
+                            {% if read == None and deleted == None and future == None %}
+                                <b>Reminder ({{ task_type_counts.reminder }})</b>
+                            {% else %} 
+                                <a href="?type=reminder" class="govuk-link govuk-link--no-visited-state govuk-link--no-underline">Reminder ({{ task_type_counts.reminder }})</a>
+                            {% endif %}
+                        </li>
+                        <li class="amp-custom-nav-bar amp-margin-right-25">
+                            {% if deleted %}
+                                <b> View deleted </b>
+                            {% else %}    
+                                <a href="?type=reminder&deleted=true" class="govuk-link govuk-link--no-visited-state govuk-link--no-underline">View deleted</a>
+                            {% endif %}
+                        </li>
+                        <li class="amp-custom-nav-bar amp-margin-right-25">
+                            {% if future %}
+                                <b> View future </b>
+                            {% else %}    
+                                <a href="?type=reminder&future=true" class="govuk-link govuk-link--no-visited-state govuk-link--no-underline">View deleted</a>
```

Comment:
The label of this link is wrong and should be "View future".

<img width="801" height="264" alt="image" src="https://github.com/user-attachments/assets/99f0dc0b-dfa5-4547-8fc4-36db0c453c49" />

Reply:
Fixed